client/components/forms/form-button/index.jsx

```diff
@@ -1,44 +1,43 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	classnames = require( 'classnames' ),
-	classNames = require( 'classnames' ),
-	omit = require( 'lodash/omit' ),
-	isEmpty = require( 'lodash/isEmpty' );
+import classNames from 'classnames';
+import { isEmpty, omit } from 'lodash';
+import React from 'react';
 
 /**
  * Internal dependencies
  */
-var Button = require( 'components/button' );
+import Button from 'components/button';
 
-module.exports = React.createClass( {
+export default React.createClass( {
 
 	displayName: 'FormsButton',
 
-	getDefaultProps: function() {
+	getDefaultProps() {
 		return {
 			isSubmitting: false,
 			isPrimary: true,
 			type: 'submit'
 		};
 	},
 
-	getDefaultButtonAction: function() {
+	getDefaultButtonAction() {
 		return this.props.isSubmitting ? this.translate( 'Saving…' ) : this.translate( 'Save Settings' );
 	},
 
-	render: function() {
-		var buttonClasses = classNames( {
-			'form-button': true
-		} );
+	render() {
+		const { children, className, isPrimary, ...props } = this.props,
+			buttonClasses = classNames( className, {
+				'form-button': true
+			} );
 
 		return (
 			<Button
-				{ ...omit( this.props, 'className' ) }
-				primary={ this.props.isPrimary }
-				className={ classnames( this.props.className, buttonClasses ) }>
-				{ isEmpty( this.props.children ) ? this.getDefaultButtonAction() : this.props.children }
+				{ ...omit( props, 'isSubmitting' ) }
+				primary={ isPrimary }
+				className={ buttonClasses }>
+				{ isEmpty( children ) ? this.getDefaultButtonAction() : children }
```

Comment:
Unrelated to this pull request since it existed previously, but I wonder:

- Is `children` valid to be specified as a `getDefaultProps` value?
- Otherwise, the more correct way to check emptiness of children is probably `React.Children.count( children ) > 0` ([docs](https://facebook.github.io/react/docs/top-level-api.html#react.children.count)), given its "opaque data structure"